python/pyspark/ml/feature.py

```diff
@@ -53,6 +53,18 @@ class Binarizer(JavaTransformer, HasInputCol, HasOutputCol):
     >>> params = {binarizer.threshold: -0.5, binarizer.outputCol: "vector"}
     >>> binarizer.transform(df, params).head().vector
     1.0
+     >>> import tempfile
```

Comment:
So as ended up being a follow up to https://github.com/apache/spark/pull/10999 we might want to simplify this a bit so it is more like an example since doctests are also (ideally) readable by users - maybe waiting for https://github.com/apache/spark/pull/11197 and then following the pattern in that PR. cc @mengxr